src/orange/surf/detail/InvolutePoint.hh

```diff
@@ -76,6 +76,7 @@ CELER_FUNCTION Real2 InvolutePoint::operator()(real_type theta) const
 {
     real_type angle = theta + a_;
     Real2 point;
+    // Efficient compiler check
```

Comment:
The comment here makes it read like you're performing a check of the compiler efficiency. Remember to write it as though it's not a note to yourself but a note to someone else:
```suggestion
    // TODO: check that compiler avoids recomputing trig functions
```